tests/e2e/scripts/pool2A.json

```diff
@@ -1,6 +1,6 @@
 {
-    "weights": "5ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518,5stake",
```

Comment:
It's likely that the account that normally would have the ibc asset due to the pre state creation txs does not have the required assets when we skip this step. This can easily be fixed by ensuring we set the receiver address to the account making the pools for when we skip the upgrade (and indirectly skip pre upgrade state creation)